### PR TITLE
Update for Fastify v5

### PR DESCRIPTION
**Pull request by gurgunday**

### CODE REVIEW

**Review by Fdawgs — APPROVED** [review]

**Comment by Fdawgs** [comment]
Flakey tests?

**Comment by gurgunday** [comment]
No seems like an issue

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,8 +17,9 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.0
     with:
       fastify-dependency-integration: true
       license-check: true
       lint: true
+      node-versions: '[16, 18, 20]'
```
Comment (mcollina) [inline .github/workflows/ci.yml]: Why?

Comment (gurgunday) [reply .github/workflows/ci.yml]: I’ll remove it, by default I was adding it to libraries repos

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -17,8 +17,9 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.1
     with:
       fastify-dependency-integration: true
       license-check: true
       lint: true
+      node-versions: '[16, 18, 20, 22]'
```
Comment (gurgunday) [inline .github/workflows/ci.yml:25]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -17,8 +17,9 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.1
```
Comment (gurgunday) [inline .github/workflows/ci.yml:20]:
```suggestion
    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v5.0.0
```

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**